http/headers/feature-policy.json

```
@@ -599,6 +599,92 @@
             }
           }
         },
+        "gamepad": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/gamepad",
+            "spec_url": "https://w3c.github.io/permissions/#gamepad",
```

@marcoscaceres Where is the HTTP Feature-Policy/Permission Policy _directive_ for  gamepad defined? This `spec_url` https://w3c.github.io/permissions/#gamepad is the permission API doc, which does not define `gamepad`. 

The reason I used this is that the other Feature-Policy directive links do link to this spec - e.g. Feature-Policy accelerometer.

---

I'm going to define it in the Gamepad spec so it's linkable. Just need to fix a couple of things first.

---

FYI, this will hopeful make things more sane https://github.com/w3c/permissions/issues/263 ... or at least I hope.

---

Thanks @marcoscaceres ! So to be clear - can I use this current URL for now?

---

Sorry for the delay! I was republishing the Gamepad spec to have the permissions link:

```suggestion
            "spec_url": "https://www.w3.org/TR/gamepad/#dfn-gamepad",
```